src/transformStateWithClones.js

```diff
@@ -7,7 +7,30 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let forCloneState = { ...state };
```

Comment:
why do we need `for` here?)

```suggestion
  let stateCopy = { ...state };
```